src/allmydata/mutable/publish.py

```diff
@@ -901,7 +901,7 @@ def _connection_problem(self, f, writer):
 
     def log_goal(self, goal, message=""):
         logmsg = [message]
-        for (shnum, server) in sorted([(s,p) for (p,s) in goal]):
+        for (shnum, server) in [(s,p) for (p,s) in goal]:
```

Comment:
Hey this looks familiar :)

Reply:
I think that's due to building branches on branches :( Or maybe it's another `sorted`? I'll fix it.